xdem/spstats.py

```diff
@@ -1003,22 +1009,273 @@ def patches_method(dh : np.ndarray, mask: np.ndarray[bool], gsd : float, area_si
     return df
 
 
-def plot_vgm(df: pd.DataFrame, fit_fun: Callable = None):
-
+def plot_vgm(df: pd.DataFrame, list_fit_fun: Optional[list[Callable[[float],float]]] = None,
+             list_fit_fun_label: Optional[list[str]] = None):
+    """
+    Plot empirical variogram, with optionally one or several model fits
+    :param df: dataframe of empirical variogram
+    :param list_fit_fun: list of function fits
```

Comment:
Just reading at the docstring, I'm not sure where those functions come from. Maybe just specify that these functions can be obtain e.g. with the function XXX.

Reply:
Clarified